src/App.jsx

```diff
@@ -14,57 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood || (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
```

Comment:
The conditional rendering here uses `selectedGood || (...)`, which means that if `selectedGood` is a falsy value (like an empty string or null), the 'No goods selected' message will be displayed. However, since `selectedGood` is initialized to 'Jam', this condition will never be true unless the state is explicitly set to a falsy value. Consider using a more explicit condition like `!selectedGood && (...)` for clarity.